Formula/gtk-doc.rb

```diff
@@ -21,41 +23,46 @@ class GtkDoc < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "2e5f324a0c3e8aac532e8946fb00cc578d18184e8fb8d247329b7fbe252054b2"
   end
 
-  depends_on "itstool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "docbook"
   depends_on "docbook-xsl"
-  depends_on "gettext"
   depends_on "libxml2"
   depends_on "python@3.9"
-  depends_on "source-highlight"
 
   uses_from_macos "libxslt"
 
+  resource "anytree" do
+    url "https://files.pythonhosted.org/packages/d8/45/de59861abc8cb66e9e95c02b214be4d52900aa92ce34241a957dcf1d569d/anytree-2.8.0.tar.gz"
+    sha256 "3f0f93f355a91bc3e6245319bf4c1d50e3416cc7a35cc1133c1ff38306bbccab"
+  end
+
+  resource "lxml" do
```

Comment:
Dependencies for experimental `mkhtml2`: 
https://gitlab.gnome.org/GNOME/gtk-doc/-/blob/master/README
> For mkhtml2 (experimental): anytree, lxml and pygments